clients/drcachesim/common/trace_entry.h

```diff
@@ -1540,6 +1540,13 @@ typedef struct _pt_data_buf_t pt_data_buf_t;
  */
 #define DRMEMTRACE_V2P_FILENAME "v2p.textproto"
 
+/**
+ * The name of the directory inside the trace directory that contains auxiliary trace
+ * files or directories that are not required to read, analyze, or schedule a trace
```

Comment:
Well, it may contain files that provide useful supplemental information for analyzing or scheduling a trace. Better to say files that do not contain trace contents.

Btw: is that where we're drawing the line then? Files that don't contain trace content are in `aux`? Do we move the existing such files to `aux` too, like the ones above?